.gitignore

```diff
@@ -1,3 +1,5 @@
+test.md
+
 # Logs
 logs
 *.log
```

Comment:
- [Optinal] Great job for adding the `test.md` and `node_modules` 👍I think it is better not to add unused files to `gitignore` to make your code clean. 👌